test/FastaToAssembly_unit_test.py

```diff
@@ -135,7 +135,14 @@ def test_import_fasta_mass_basic_file_None_mcl_key(tmp_path):
     _test_import_fasta_mass_file(tmp_path, {'min_contig_length': None})
 
 
-def _test_import_fasta_mass_file(tmp_path, params_root):
+def test_import_fasta_mass_with_obj_meta(tmp_path):
```

Comment:
I assume there are tests that don't include metadata?

Reply:
yea. A few existing tests, `test_import_fasta_mass_basic_file_no_mcl_key` and `test_import_fasta_mass_basic_file_None_mcl_key` do not include metadata.

Reply:
:+1: